webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1233,6 +1282,118 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
             zipOut.putNextEntry(null, parameters);
             zipOut.write(join(stripNewlines(scrambleRequest.getAllScrambles()), "\r\n").getBytes());
             zipOut.closeEntry();
+                
+            parameters.setFileNameInZip("Internationalization/");
+            
+            ArrayList<String> availableLanguages = new ArrayList<String>();
+            // TODO
+            // Jeremy, this is where the iteration over files can let internationalization easier to handle.
+            // The following lines must be replaced.
+            availableLanguages.add("MessagesBundle.properties");
+            availableLanguages.add("MessagesBundle_en_US.properties");
+            availableLanguages.add("MessagesBundle_pt_BR.properties");
+            
+            for (String item : availableLanguages) {
+                String[] temp = item.split("_|\\.");
+                
+                if (temp.length != 4){
+                    continue;
+                }
+                
+                String language = temp[1];
+                String country = temp[2];
+                
+                // fewest moves regular sheet
+                pdfFileName = "Internationalization/"+language+"_"+country+"_"+safeTitle+".pdf";
```

Comment:
These are PDFs, so I believe they belong in the pdf folder. I also think it's weird to capitalize the folder name when non of our other folders are capitalized. How about something like `"pdfs/i18n/"+language+"_"+country+"_"+safeTitle+".pdf"` instead?

Reply:
I agree with you about the capitalization and folder placing. I don't think i18n is a good name, since is very common among programmers, but not to non programmers. I'm afraid the UI might get less user friendly.

Reply:
I'm considering pdf/translations. Is intuitive and short enough.